File: q/c/submissions/Laterals/problem1.c
#include<stdio.h>
#include<math.h>
int main()
{
   int a,flag=0,i;
   scanf("%d",&a);
   
   for(i=2; i<=a/2; i++)
   {
      if(a%i==0)
      {
          flag=1;
      }
      else
      {
          flag=0;
      }
   }
   if(flag==0)
      printf("Yes");
   else
      printf("No");
}

File: round 1/ASTRO/problem1.java
import java.util.*;
class problem1{
public static void main(String args[]){
Scanner sc=new Scanner(System.in);
long a;
a=sc.nextLong();
long k=(long)(Math.sqrt(a));
int flag=0;
for(long i=2;i<=k;i++){
if(a%i==0)
flag=100;
}
if(flag==100)
System.out.println("No");
else
System.out.println("Yes");
}
}